Detectors/MUON/MCH/Workflow/src/cru-page-reader-workflow.cxx

```diff
@@ -168,6 +168,7 @@ class FileReaderTask
     /// Get the input file and other options from the context
     LOG(INFO) << "initializing file reader";
     mFrameMax = ic.options().get<int>("nframes");
+    mTimeFrameMax = ic.options().get<int>("max-time-frame");
```

Comment:
more a note to self than a comment to this PR : at some point we should uniformize (at least within muon) the names of some workflow options. e.g. for digit [reader/writer](https://github.com/AliceO2Group/AliceO2/tree/dev/Detectors/MUON/MCH/DevIO/Digits) I've used `--first-tf` and `--max-nof-tfs` for a similar purpose